config/owasp/suppressions.xml

```diff
@@ -1,5 +1,20 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.1.xsd">
+
+  <suppress>
+    <notes><![CDATA[postgresql before versions 10.4, 9.6.9 is vulnerable in the adminpack extension, the pg_catalog.pg_logfile_rotate() function doesn't follow the same ACLs than pg_rorate_logfile. If the adminpack is added to a database, an attacker able to connect to it could exploit this to force log rotation]]></notes>
```

Comment:
a comment on why this got suppressed would be nice

Reply:
the suppression itself explains it in detail: notes text copy from web + gav regex to which dependency is a culprit + cve - link to detailed info

Reply:
It explains why there is a problem with this library, not why you've decided to ignore this warning.